src/Libraries/CoreNodes/Math.cs

```diff
@@ -673,13 +673,13 @@ public static long Factorial(long number)
         /// <summary>
         ///     Boolean XOR: Returns true if and only if exactly one of the inputs is true.
         /// </summary>
-        /// <param name="a">A boolean.</param>
-        /// <param name="b">A boolean.</param>
+        /// <param name="bool">First boolean.</param>
+        /// <param name="otherBool">Second boolean.</param>
```

Comment:
would probably use other Boolean like in the method inputs.

Reply:
this was discussed by Autodesk, int and bool was used in inputs and outputs names instead of integer and boolean, we use integer and boolean in descriptions

Reply:
yea that wasnt really what i meant. I was just saying that i would use `Other boolean` in the description instead of `Second boolean` i think we are using `Other boolean` in more scenarios.

Reply:
ok, addressed in cbb717a